spec/facades/book_facade_spec.rb

```diff
@@ -25,11 +25,17 @@
 
           expect(actual.length).to eq(10)
           expect(actual[0].class).to eq(BookPoro)
-          expect(actual[0].g_id).to eq('MRm8DwAAQBAJ')
-          expect(actual[0].title).to eq('Silver Sparrow')
-          expect(actual[0].authors).to eq(['Tayari Jones'])
```

Comment:
Did the cassettes not get sent in the pull request? we will have to be careful when pulling and pushing not to overwrite them or everyones tests will fail differntly.

Reply:
@LawrenceWhalen I think this is on me here - I didn't realize that cassettes wouldn't be shared in the github repo, and I ended up resetting them all in order to get tests passing (and the response data specifically for pg 2 of the book search results was different)